### ISSUE TITLE
Openstack docset

### ISSUE THREAD

**Issue by dudymas** [Kapeli/Dash-User-Contributions#355]
Hey. I've made the docset for openstack and it literally is about 110 MB... just over the 100MB limit. If you include the pdf's, it goes to 140MB or so.

Can we work out a way to host the docset somewhere else?

Thanks.

**Comment by Kapeli** [comment]
Sure. Upload the docset anywhere you can (Dropbox) and submit a pull request that sets up the directory structure without the docset. I'll do the rest.

**Comment by marcblabla** [comment]
Hi

What ever happened to this? 
Because I can't seem to find the OpenStack docset anywhere.

**Comment by Kapeli** [comment]
@marcblabla I've never received a pull request for the docset.

**Comment by marcblabla** [comment]
@Kapeli Well that explains it :-). Do you know anywhere else I can get it?

**Comment by Kapeli** [comment]
No, sorry.

**Comment by dudymas** [comment]
Ahoy, I know this is dead, but wanted to explain a bit more. I had this working back in March on openstack juno, but quickly found:
1) I had to utterly roll my own doc builder
2) still found pages that broke Dash and needed hand editing
3) The search indexes were no where near as good as the actual hosted documentation
4) oh and look at that, kilo is already out and way better than juno

I just wouldn't have the free time to keep the docset up and working, and if I did, it would be obsolete. I thought about killing this issue outright... but was hoping maybe I'd find someone else saying they'd found a way to do this already. As it is... I find it pretty annoying that the openstack docs are always online and usually fraught with slow loading times. And they're so crucial when you are first learning openstack.
